jwt.go

```diff
@@ -84,13 +274,20 @@ func (t *Token) SigningString() (string, error) {
 // keyFunc will receive the parsed token and should return the key for validating.
 // If everything is kosher, err will be nil
 func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	return ParseWithClaims(tokenString, keyFunc, &Claims{})
```

Comment:
I don't think this works.  The default type should still be the map type.  The main reason for this is, the standard `Claims` type doesn't include any user properties and is probably only useful when wrapped with a custom type.  The map type, while less clean, will capture all the fields.